reference/functions/string_replace.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: string_replace
+published: true
+tags: [reference, data functions, functions, string_replace]
+---
+
+[%CFEngine_function_prototype(string, match, replacement)%]
+
+**Description:** In a given string, replaces a substring with another string.
+
+[%CFEngine_function_attributes(string, match, replacement)%]
+
+Reads a string from left to right, replacing the occurences of the second
+argument with the third argument in order.
+
+All characters in the string to replace in, the substring to match for and the
+replacement are read literally. This means that `.`, `*`, `\` and similar
+characters will be read and replaced as they are. If you are looking for more
+advanced replace functionality, check out `regex_replace()`.
+
+Example:
```

Comment:
Example, output, history, and see also should be bolded like Description is above.

Reply:
Output doesn't seem to be bolded in a lot of the other reference entries?

Reply:
Seems like it is inconsistent. I'll add it anyway!